lib/blueprinter/field.rb

```diff
@@ -18,9 +18,34 @@ def extract(object, local_options)
     end
 
     def skip?(field_name, object, local_options)
+      return true if unless_callable && unless_callable.call(field_name, object, local_options)
       return true if if_callable && !if_callable.call(field_name, object, local_options)
 
-      unless_callable && unless_callable.call(field_name, object, local_options)
+      false
+    end
+
+    def clone
```

Comment:
This is necessary to ensure we don't mutate parent fields by adding conditions to them